content/applications/sales/crm/track_leads/marketing_attribution.rst

```diff
@@ -0,0 +1,142 @@
+generation, attribution, won rate, and more.
+
+Leads Analysis dashboard
+========================
+
+Begin by navigating to the :guilabel:`Leads Analysis` dashboard by going to :menuselection:`CRM app
+--> Reporting --> Leads`.
+
+.. tip::
+   Alternatively, reports can also be run on the :menuselection:`CRM app --> Leads` dashboard or the
+   :menuselection:`CRM app --> Sales --> My Pipeline` dashboard (in case the :guilabel:`Leads`
+   dashboard is not available); both dashboards contain the necessary :guilabel:`Filters` and
+   :guilabel:`Group By` criteria to run an attribution report.
+
+.. seealso::
+   - :doc:`Convert leads into opportunities <../acquire_leads/convert>`
+   - :doc:`Create leads <../acquire_leads/email_manual>`
+
+.. image:: marketing_attribution/reporting-tab-and-leads.png
+   :align: center
+   :alt: Open the CRM app and click on the Reporting tab at the top of the page, then click Leads.
+
+The :icon:`fa-area-chart` :guilabel:`(Graph)` view will be shown by default, with :guilabel:`Active
+or Inactive` and :guilabel:`Created on: YYYY` filters active in the :guilabel:`Search...` bar. The
+graph visualization displays the number of leads generated, by month and by sales team, with each
+sales team attributed to its own color per bar (month) shown.
+
+Switch the view to the :icon:`oi-view-list` :guilabel:`(List)` option, by clicking the respective
+icon located at the top-right of the dashboard.
+
+.. image:: marketing_attribution/list-view-button.png
+   :align: center
+   :alt: Click the button with four horizontal lines on the top right of the Leads Analysis page.
+
+Add UTM parameters
+==================
+
+*Urchin Tracking Modules (UTMs)* are snippets of text embedded in URLs that allow for data about
+visitors to be tracked. This includes parameters relating to how a visitor reached the link, such as
+the type of website visited and marketing campaign the visit came from.
+
+Odoo can use these UTMs as parameters in the marketing attribution report to track the metrics and
+performance of marketing campaigns.
+
+Create UTMs
+-----------
+
+The :doc:`link tracker <../../../websites/website/reporting/link_tracker>` in Odoo can be used to
+create and configure UTMs.
+
+UTMs can also be automatically generated by :doc:`Email Marketing
+<../../../marketing/email_marketing>` and :doc:`Marketing Automation
+<../../../marketing/marketing_automation>` apps.
+
+The UTM parameters used in a marketing attribution report are *Medium*, *Source*, and *Campaign* in
+descending order of coverage.
+
+- *Medium* is the UTM with the widest coverage, such as social media.
+- *Source* is more narrow and covers parameters such as the specific social media platform.
+- *Campaign* is the narrowest and can track specific marketing campaigns.
+
+Create reports
+==============
+
+To start creating a report, click the :icon:`fa-caret-down` :guilabel:`(down caret)` to the right of
```

Comment:
```suggestion
To start creating a report, click the :icon:`fa-caret-down` :guilabel:`(down arrow)` to the right of
```
I know this hasn't been "set in stone" or anything, but I am always "in the camp" of making things as universally-understood as possible. Especially with highly technical jargon, such as 'caret' (arrow) or 'modal' (pop-up)...I just feel like the average reader wouldn't exactly know that term off-the-cuff. In other words, they may actually look for a 'carrot' icon (despite the spelling difference). All that said, I think it's more useful to the reader, and the doc, in general, by using the term "arrow" in lieu of "caret" for those reasonings. What you did was *technically* spot-on, 100% correct -- I'm just suggesting this difference to appease the otherwise not-as-tech-savvy reader.